services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -219,6 +221,54 @@ testRichInfo = do
     liftIO $ scimUserId scimStoredUser' `shouldBe` scimUserId scimStoredUser
     probeUser (scimUserId scimStoredUser) richInfo'
 
+-- | Create a user implicitly via saml login; remove it via brig leaving a dangling entry in
+-- @spar.user@; create it via scim.  This should to work despite the dangling database entry.
+testScimCreateVsUserRef :: HasCallStack => TestSpar ()
+testScimCreateVsUserRef = do
+    (tok, (_ownerid, _teamid, idp)) <- registerIdPAndScimToken
+    (usr, uname) :: (Scim.User.User SparTag, SAML.UnqualifiedNameID)
+        <- randomScimUserWithSubject
+    let uref   = SAML.UserRef tenant subj
+        subj   = either (error . show) id $ SAML.mkNameID uname Nothing Nothing Nothing
+        tenant = idp ^. SAML.idpMetadata . SAML.edIssuer
+
+    uid <- createViaSaml idp uref
+    samlUserShouldSatisfy 'a' uref isJust
```

Comment:
Is there a reason for a char instead of `HasCallStack` on the internal function?

Reply:
Couldn't get `HasCallStack` to do the trick.  I never got to the code line that I was interested in.  (Wanna try?)

Reply:
(I'll resolve this since I don't think it's important enough, but feel free to re-open.)

Reply:
> (Wanna try?)

Nah.